packages/pwa-kit-dev/package.json

```diff
@@ -61,7 +61,7 @@
     "babel-plugin-dynamic-import-node-babel-7": "^2.0.7",
     "babel-plugin-formatjs": "10.2.20",
     "chalk": "^4.1.2",
-    "commander": "^8.3.0",
+    "commander": "^9.3.0",
```

Comment:
I kept this library upgrade from the initial spike. I noticed PWA Kit team's 3PP approval for this lib is only for v2.20.3 (?!), so I'll look into requesting approval for v9.

Reply:
Here's my approval request: https://gus.lightning.force.com/lightning/r/ADM_Third_Party_Software__c/a0qEE0000005Si1YAE/view